tests/unittests/ExpressionTests.cpp

```diff
@@ -1239,4 +1239,69 @@ endmodule
     REQUIRE(diags.size() == 2);
     CHECK(diags[0].code == diag::BadRangeExpression);
     CHECK(diags[1].code == diag::BadRangeExpression);
-}
\ No newline at end of file
+}
+
+std::string testStringLiteralsToByteArray(const std::string& text) {
+    const auto& fullText = "module Top; " + text + " endmodule";
+    auto tree = SyntaxTree::fromText(string_view(fullText));
+
+    Compilation compilation;
+    compilation.addSyntaxTree(tree);
+    NO_COMPILATION_ERRORS;
+
+    const auto& module = *compilation.getRoot().topInstances[0];
+    if (!tree->diagnostics().empty())
```

Comment:
The NO_COMPILATION_ERRORS macro will already check that there are no diagnostics, so this bit of code here is unnecessary.

Reply:
I removed the redundant code as suggested.